tests/sentry/integrations/slack/notifications/notifications/test_organization_request.py

```diff
@@ -0,0 +1,58 @@
+from sentry.models import NotificationSetting, OrganizationMember
```

Comment:
Why put this in a new directory instead of with the other slack notif tests?

Reply:
@ceorourke My understanding is that the location of tests should match the file of the main entity being tested

Reply:
On wait, this is not supposed to be in the slack directory...good catch!